editor.planx.uk/src/@planx/components/MultipleFileUpload/Modal.tsx

```diff
@@ -60,3 +72,78 @@ export const FileTaggingModal = (props: FileTaggingModalProps) => {
     </Dialog>
   );
 };
+
+interface SelectMultipleProps extends SelectProps {
+  name: string;
+  fileList: FileList;
+}
+
+const SelectMultiple = (props: SelectMultipleProps) => {
+  const { name, fileList } = props;
+  const [tags, setTags] = React.useState<string[]>([]);
+
+  const handleChange = (event: SelectChangeEvent<typeof tags>) => {
+    const {
+      target: { value },
+    } = event;
+    setTags(
+      // On autofill we get a stringified value.
+      typeof value === "string" ? value.split(",") : value
+    );
+  };
+
+  return (
+    <FormControl key={name} sx={{ display: "flex", flexDirection: "column" }}>
+      <InputLabel id="select-mutliple-file-tags-label">
+        What does this file show?
+      </InputLabel>
+      <Select
+        id="select-multiple-file-tags"
+        labelId="select-multiple-file-tags-label"
```

Comment:
I would guess both of these need to be unique to make React (and a11y tests) happy, e.g. ``id={`select-multiple-file-tags-${name}`}``

Reply:
updated!